media_kit_core_video/windows/angle_surface_manager.cc

```diff
@@ -56,9 +57,11 @@ void ANGLESurfaceManager::SwapBuffers() {
 
 void ANGLESurfaceManager::MakeCurrent(bool value) {
```

Comment:
Now that releasing context is not required (based on your last comment), let's remove the argument from this method.

And only use it to acquire context.

```suggestion
void ANGLESurfaceManager::MakeCurrent() {
```

Thus, remove entirety of this (if it goes well):

https://github.com/alexmercerind/media_kit/blob/083f128876f4a156a61c70a73c93558b808e3c1f/media_kit_core_video/windows/video_output.cc#L147-L158

Reply:
Ill test it for sure when I can

Reply:
after half an hour of the two side by side, it's best to keep it as is, it's quite a bit more stable? it flickers left often.